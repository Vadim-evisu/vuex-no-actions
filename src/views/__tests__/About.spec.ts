import { shallowMount } from '@vue/test-utils';
import About from '@/views/About.vue';

it('sanity', () => {
  expect(true).toBe(true);
});
describe('About view', () => {
  const wrapper = shallowMount(About);
  it('matches snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
