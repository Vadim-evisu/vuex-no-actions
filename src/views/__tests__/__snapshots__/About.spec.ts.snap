// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About view matches snapshot 1`] = `
<div>
  <h1>
    This is an about page
  </h1>
</div>
`;
